webdriver-spec.html

```diff
@@ -5780,118 +5774,19 @@
  <li><p>Let <var>result</var> be the result of calling <a>execute a function body</a>,
   with arguments <var>body</var> and <var>arguments</var>.
 
- <li><p>Wait for <a>execute a function body</a> to return
-  or for <a>session script timeout</a> milliseconds to expire,
-  whichever happens sooner.
+ <li><p>Let <var>promise</var> be a new Promise.
```

Comment:
`Promise` should be defined as a dependency and link to https://tc39.github.io/ecma262/#sec-promise-constructor